registry/txt_test.go

```diff
@@ -451,6 +471,7 @@ func testTXTRegistryApplyChanges(t *testing.T) {
 	t.Run("With Templated Suffix", testTXTRegistryApplyChangesWithTemplatedSuffix)
 	t.Run("With Suffix", testTXTRegistryApplyChangesWithSuffix)
 	t.Run("No prefix", testTXTRegistryApplyChangesNoPrefix)
+	t.Run("No prefix Metadata Only", testTXTRegistryApplyChangesNoPrefixMetadataOnly)
```

Comment:
Call this just "Metadata only". The results should be the same for with-prefix and with-suffix and the code should test those cases as well.